### ISSUE TITLE
Visdom blank blue screen, when setting layout title plotlyplot

### ISSUE THREAD

**Issue by ktagowski** [facebookresearch/visdom#565]
**Bug Description**
I got a blank blue screen of wisdom (0.1.8.8) when I set the title of the plotly annotated heatmap.

**Reproduction Steps**
```python
fig = ff.create_annotated_heatmap(z, x=x, y=y, colorscale='Viridis', showscale=True)
fig.layout.title='Test title'

viz.plotlyplot(fig)
```

**Client logs:**
```javascript
TypeError: n.title.match is not a function
    at main.js?v=4316a77ded15b92b6e884b60b72f6513:35
    at Array.map (<anonymous>)
    at t.value (main.js?v=4316a77ded15b92b6e884b60b72f6513:35)
    at h (react-dom.min.js?v=950495cc51ccb90612cf0fe0bb44f8f3:130)
    at beginWork (react-dom.min.js?v=950495cc51ccb90612cf0fe0bb44f8f3:134)
    at d (react-dom.min.js?v=950495cc51ccb90612cf0fe0bb44f8f3:158)
    at f (react-dom.min.js?v=950495cc51ccb90612cf0fe0bb44f8f3:159)
    at g (react-dom.min.js?v=950495cc51ccb90612cf0fe0bb44f8f3:159)
    at t (react-dom.min.js?v=950495cc51ccb90612cf0fe0bb44f8f3:167)
    at x (react-dom.min.js?v=950495cc51ccb90612cf0fe0bb44f8f3:166)
```

**Server logs:**

> INFO:root:Opened visdom socket from ip: 127.0.0.1
> INFO:tornado.access:200 POST /events (127.0.0.1) 1.59ms

**Comment by pringshia** [comment]
@ktagowski This issue was fixed with #561 

You could either try installing visdom from source for now (`pip install -e .` in source dir), or wait until a new release is cut (cc/ @JackUrb )

**Comment by ktagowski** [comment]
@pringshia Thank you very much for your help. I haven't noticed that this commit solves it.
